MatchmakingServer/Program.cs

```diff
@@ -0,0 +1,111 @@
+using System.Text.Json.Serialization;
+
+using H2MLauncher.Core.Interfaces;
+using H2MLauncher.Core.Services;
+using H2MLauncher.Core.Utilities;
+
+using MatchmakingServer;
+using MatchmakingServer.Authentication;
+using MatchmakingServer.SignalR;
+
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+
+var builder = WebApplication.CreateBuilder();
```

Comment:
I assume it is a IWebBuilder ?

Reply:
Yup. It's default code from the template